Sources/Helpers/Image+.swift

```diff
@@ -0,0 +1,9 @@
+import SwiftUI
+
+extension Image {
+    /// Loads an image from the Warp bundle.
+    /// - Parameter named: The rawValue of the icon from `Warp Assets`.
+    public static func warpIcon(named name: String) -> Image {
```

Comment:
isn't it a little better to have this namespaced to  `Warp`?
ex: `Warp.icon(name: String)`
it's ok it this way as well, but it does seem that all other endpoins Warp provides are under Warp namespace and not extensions to native types

Reply:
Poor Xcode does not know what to use without more context 😅

<img width="902" alt="Screenshot 2024-11-08 at 15 38 15" src="https://github.com/user-attachments/assets/5e92331d-fd3a-4e75-aeef-639431acddb4">

So I will leave it like this since we have other public extensions like cardBackground and roundedCorners and the like.